editor/extension.ts

```diff
@@ -178,6 +178,39 @@ namespace pxt.editor {
                 block.setAttribute("type", "SpriteKindLegacy")
             });
         }
+
+        if (pxt.semver.strcmp(pkgTargetVersion || "0.0.0", "0.18.8") < 0) {
```

Comment:
Does this need to be 18.9?

Reply:
Yeah I'll need to bump the minor to the current released one, I'm gonna wait till https://github.com/microsoft/pxt-arcade/pull/2085 is merged to update this and do a separate bump, that's a big enough change on it's own / easier to track things in future ~

Reply:
oo nice cool cool

Reply:
merged!